### PR TITLE
Project Cluedo Lisa Sturegard

### PR DESCRIPTION
**Pull request by LisaStur**

### CODE REVIEW

File: code/clue.js [hunk 1]
```diff
@@ -1,59 +1,261 @@
...
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.jpg'
+}
+
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.jpg'
+
```
Comment (ericaglimsholt) [inline code/clue.js:176]:
```suggestion
```

File: code/index.html [hunk 2]
```diff
@@ -5,32 +5,36 @@
...
-        <p id="killerOccupation"></p>
         <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
       </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
+      <article class="weapon-deck" id="weaponCard" onclick="pickWeapon()">
+        <h2>WEAPON</h2>
         <p id="weaponName"></p>
+        <img id="weaponImage" src="" height="100" />
```
Comment (ericaglimsholt) [inline code/index.html:24]: Try add the styling in the css instead of html :)

File: code/style.css [hunk 3]
```diff
@@ -1,58 +1,110 @@
...
-  font-weight: 300;
-  background-color: #fff;
+  font-size: 25px;
+}
+
+h1 {
+font-family: 'Bowlby One SC', cursive;
+font-size: 100px;
+color: cornsilk;
+margin: 0;
```
Comment (ericaglimsholt) [inline code/style.css:18]: Indent one step

**Review by ericaglimsholt — APPROVED** [review]
Great work Lisa! 
I couldn't really find anything to comments about. You did a great work! Keep it up :D

File: code/index.html (hunk 2 above)
Comment (LisaStur) [reply code/index.html:24]: Ah, sure! Big thanks! :)

**Pull request closed without merging**